flask_admin/contrib/sqla/fields.py

```diff
@@ -254,3 +253,27 @@ def get_pk_from_identity(obj):
     # TODO: Remove me
     cls, key = identity_key(instance=obj)
     return u':'.join(text_type(x) for x in key)
+
+
+def get_obj_pk(obj, pk):
+    """
+    get and format pk from obj
+    :rtype: text_type
+    """
+
+    if isinstance(pk, tuple):
+        return u':'.join(text_type(getattr(obj, k)) for k in pk)
```

Comment:
I can't accept this - if you need multiple items, please use tuple. PK can be any value and it can have ':' in it.